server/conf/cms/secrets.sample.py

```diff
@@ -249,6 +260,13 @@
 
 _LOGO = _PORTAL_LOGO                # Default Portal Logo.
 
+########################
+# FAVICON
+
+_FAVICON = {
+    "img_file_src": "site_cms/img/favicons/favicon.ico"
+}
+
```

Comment:
This is recently new to CMS. Adding this prevents a CMS load failure on fresh Portal-n-CMS setup.